app/features/notes_generator/tests/test_tools.py

```diff
@@ -0,0 +1,49 @@
+import pytest
```

Comment:
The `test_core.py` file actually tests the tool itself. This file is not required for not generating redundancy.